README.md

```diff
@@ -28,15 +28,7 @@ You can install the package via composer:
 composer require alymosul/laravel-exponent-push-notifications
 ```
 
-You must install the service provider:
```

Comment:
You should leave this for people using older versions of laravel

Reply:
Done :)